packages/e2e/utils/test-user.ts

```diff
@@ -0,0 +1,63 @@
+import { destroyInbox, createInbox } from './mail';
+import { createUser, confirmUserPassword, destroyUser } from './aws';
+
+export interface TestUser {
+  address: string;
+  id: string;
+  cognitoUser: string;
+}
+
+export const createTestUser = async (): Promise<TestUser> => {
+  console.warn('Creating a new test user...');
+
+  let userInbox: { id: string; address: string };
```

Comment:
These can be consts, I think

Reply:
hm nope. that's just the variable declaration. the assignment happens inside the try/catch block

Reply:
Oh, didn't notice - my bad! Resolving!